db/external_sst_file_ingestion_job.cc

```diff
@@ -103,12 +103,14 @@ Status ExternalSstFileIngestionJob::Prepare(
         // Original file is on a different FS, use copy instead of hard linking
         status = CopyFile(env_, path_outside_db, path_inside_db, 0,
                           db_options_.use_fsync);
```

Comment:
will the compiler complain if we put `f.copy_file=true` after this `CopyFile` and the lower one, and give it no default value? It's slightly clearer and less error-prone to me to initialize it only in the places where we know for certain what its value should be (and that prevents us from accidentally using the default value when code paths change in the future). But that's just my personal preference, will leave it up to you whether to address.

Reply:
Done.